testing/unittests/metrics/MetricFrameworkTests.cpp

```diff
@@ -312,16 +312,22 @@ class MetricFrameworkTests : public CppUnit::TestFixture
 
     void Test_histogram_metric()
     {
-        std::vector<BucketDef> bucketDefs = {{"le 2", 2}, {"le 4", 4}, {"le 8", 8} };
+        //std::vector<BucketDef> bucketDefs = {{"le 2", 2}, {"le 4", 4}, {"le 8", 8} };
+        std::vector<__uint64> bucketDefs = {2, 4, 8};
         std::shared_ptr<HistogramMetric> pHistogram = std::make_shared<HistogramMetric>("requests.dist", "description", SMeasureCount, bucketDefs);
 
         std::vector<__uint64> values;
 
+        printf("Validating number of returned buckets\n");
```

Comment:
Possibly better is to use DBGLOG.  If you run with the -v option the logging is traced to stdout